src/index.ts

```diff
@@ -1,124 +1,38 @@
-import { Beautifier, Language } from "unibeautify";
+import { Beautifier, Language, BeautifierBeautifyData } from "unibeautify";
 const prettydiff = require("prettydiff2");
-
+import options from "./options";
 const pkg = require("../package.json");
-
 export const beautifier: Beautifier = {
   name: "Pretty Diff",
   package: pkg,
   options: {
```

Comment:
What about passing the imported `options` variable into this `options` property?